middleware/src/handlers/handlers.go

```diff
@@ -0,0 +1,91 @@
+// Package handlers implements an api for the bitbox-wallet-app to talk to.
+package handlers
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	middleware "github.com/digitalbitbox/bitbox-base/middleware/src"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+	//"github.com/sirupsen/logrus"
+)
+
+// Middleware provides an interface to the middleware package.
+type Middleware interface {
+	Start() <-chan *middleware.SampleInfo
+}
+
+// Handlers provides a web api
+type Handlers struct {
+	Router   *mux.Router
+	clients  map[*websocket.Conn]bool
+	upgrader websocket.Upgrader //upgrader takes an http request and upgrades the connection with its origin to websocket
+	//log        log.Logger
+	middleware Middleware
+	//TODO(TheCharlatan): In future this event should have a generic interface (thus only containing raw json)
+	middlewareEvents <-chan *middleware.SampleInfo
+}
+
+// NewHandlers returns a handler instance.
+func NewHandlers(middlewareInstance Middleware) *Handlers {
+	router := mux.NewRouter()
+
+	handlers := &Handlers{
+		middleware: middlewareInstance,
+		Router:     router,
+		clients:    make(map[*websocket.Conn]bool),
+		upgrader: websocket.Upgrader{
+			CheckOrigin: func(r *http.Request) bool {
+				return true
+			},
+		},
+		//log: log.NewLogger(),
+	}
+	//log.WithField("group", "Handlers")
+	handlers.Router.HandleFunc("/", handlers.rootHandler).Methods("GET")
+	handlers.Router.HandleFunc("/ws", handlers.wsHandler)
+
+	handlers.middlewareEvents = handlers.middleware.Start()
+	go handlers.echo()
+	return handlers
+}
+
+// rootHandler provides an endpoint to indicate that the middleware is online and able to handle requests. TODO(TheCharlatan): Define a better error-response system. In future, this should be the first step in an authentication procedure.
```

Comment:
The `TODO` here is useful, but a bit hard to notice. Perhaps it should be on its own line?

Reply:
Move to own line.